pkg/catalogv2/system/system.go

```diff
@@ -257,17 +257,21 @@ func (m *Manager) Remove(namespace, name string) {
 	}
 }
 
-// install tries to install a new version of a chart. If the exact version is provided, it will try to install it
-// otherwise it will try to install the latest version available. If a release with the version to be installed is already installed,
-// or it's pending install, upgrade or rollback this does nothing.
+// install tries to install a new version of a chart. If the exact version is provided, it will try to install it;
+// otherwise it will return an error without altering existing state. If a release with the version to be installed is
+// already installed, or is pending install, upgrade or rollback, this does nothing.
```

Comment:
This update makes it sound like the function always returns an error if the exact version is not provided.
```suggestion
// install tries to install a new version of a chart. If the exact version is provided, it will try to install it.
// If minVersion is provided, it will try to install it only if the current version is earlier than minVersion.
// otherwise, it will try to install the latest version available. if a release with the version to be installed is
// already installed, or is pending install, upgrade or rollback, this does nothing.
```

Reply:
Thanks, included this suggestion with a few tweaks as discussed offline.